seed_db.sh

```diff
@@ -70,164 +70,230 @@ else
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('Pride and Prejudice', 'Jane Austen', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 4, '[\"GERMAN\", \"ENGLISH_UK\"]');"
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('To Kill a Mockingbird', 'Harper Lee', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 3, '[\"GERMAN\", \"ENGLISH_UK\", \"PORTUGUESE\", \"DUTCH\"]');"
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('The Great Gatsby', 'F. Scott Fitzgerald', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 4, '[\"ENGLISH_US\"]');"
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('1984', 'George Orwell', 'https://www.youtube.com/watch?v=Da-2h2B4faU&t=4s', 2, '[]');"
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "
+            USE planet-read; 
+            INSERT INTO stories 
+                (title, description, youtube_link, level, translated_languages) 
+            VALUES 
+                ('East of Eden', 'John Steinbeck', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 4, '[\"GERMAN\", \"ENGLISH_UK\"]'),
+                ('War and Peace', 'Leo Tolstoy', 'https://www.youtube.com/watch?v=Da-2h2B4faU&t=4s', 1, '[\"GERMAN\", \"POLISH\"]'),
+                ('A Tale of Two Cities', 'Charles Dickens', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 3, '[\"MANDARIN\", \"ENGLISH_UK\"]'),
+                ('Pride and Prejudice', 'Jane Austen', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 4, '[\"GERMAN\", \"ENGLISH_UK\"]'),
+                ('To Kill a Mockingbird', 'Harper Lee', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 3, '[\"GERMAN\", \"ENGLISH_UK\", \"PORTUGUESE\", \"DUTCH\"]'),
+                ('The Great Gatsby', 'F. Scott Fitzgerald', 'https://www.youtube.com/watch?v=DHyUYg8X31c', 4, '[\"ENGLISH_US\"]'),
+                ('1984', 'George Orwell', 'https://www.youtube.com/watch?v=Da-2h2B4faU&t=4s', 2, '[]');
+        "
+        
+        echo "Finished generating stories..." 
+
     fi 
-    
 
     # story_translations
     docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; ALTER TABLE story_translations AUTO_INCREMENT = 1;"
 
+    docker exec -it planet-read_db_1 mysql -u root -proot -e "
+        USE planet-read; 
+        INSERT INTO story_translations 
+            (story_id, language, stage, translator_id) 
+        VALUES 
+            (1, 'GERMAN', 'TRANSLATE', 6),
```

Comment:
nit: the spacing and organization here is a little weird. can you ascend by story_id and remove spaces?
![image](https://user-images.githubusercontent.com/15113249/127957133-42b80022-5af6-4ce4-9a8d-224b5dfbe03f.png)